esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2093,27 +2093,24 @@ bool Cws_accessEx::onResourceDelete(IEspContext &context, IEspResourceDeleteRequ
                 if(prefix && *prefix)
                     namebuf.insert(0, prefix);
 
-                try
-                {
-                    secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
-                }
-                catch(IException* e)
-                {
-                    me->append(*e);
-                }
+                secmgr->deleteResource(rtype, namebuf.str(), basednReq->getBasedn(), context.querySecureContext());
```

Comment:
This means that if any of the deletes fail, the rest of the deletes will not be done.  I suspect they should still happen.

Reply:
@ghalliday You are correct and I opened https://track.hpccsystems.com/browse/HPCC-29398 to address that problem.
My intent with this PR was to restore the behavior to what it has always been in the past, ensuring user can move on. Let me know what you want me to do

Reply:
@ghalliday I went ahead with the change you suggested (continuing on error), which differs from the legacy behavior of 8.10 and before. Note that the called procedure "secmgr->deleteResource()" does not throw, so need for those catches

Reply:
@ghalliday You are correct, the catch blocks were artifacts from a previous PR that are no longer needed